[WIP] Use custom application name in tray menu (Kepka!)

minlexx opened the pull request:
TODO:
- other languages? use lang string with application name as parameter
- squash commits....

Commit:
Use custom application name in tray popup menu

Commit:
Use custom application name in tray popup menu

Commit:
tabs!

Commit:
tabs!

crackedmind commented:
:white_check_mark: [Build kepka 1.0.66-omhbptth completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.66-omhbptth) (commit https://github.com/procxx/kepka/commit/89cc6c69ec by @minlexx)

ForNeVeR commented on Telegram/SourceFiles/mainwindow.cpp:
```diff
@@ -166,14 +166,14 @@ void MainWindow::firstShow() {
 
 	if (isLinux) {
-		trayIconMenu->addAction(lang(lng_open_from_tray), this, SLOT(showFromTray()))->setEnabled(true);
+		trayIconMenu->addAction(lang(lng_open_from_tray) + QCoreApplication::applicationName(), this, SLOT(showFromTray()))->setEnabled(true);
```
That's not a good idea. There're languages that use a different word order (e.g. in Azerbaijani that would be something like "Kepka/Telegram çıxın"). I suggest that we use the text templates here (e.g. "Quit {app_name}", like [here](https://github.com/minlexx/kepka/blob/34e3ac75e189da9bf84cd0588cd40afbf2185234/Telegram/Resources/langs/lang.strings#L99)) and use string format to prepare the resulting value.

minlexx replied:
Yep, will fix later when I have time. Need to figure out how this translation system works... I see it automatically generates some .cpp and .h files

crackedmind committed:
Update build instructions [skip ci]

leha-bot committed:
Move out the hardcoded URLs (changelog, about)

leha-bot committed:
Get rid of usused file freedesktop.org.xml

It is ambigious and don't need.

anatoly-spb committed:
Made to compile by MSVC 2015/2017 32 bit compilers (#101)

Patches by @anatoly-spb :
1. Get rid of -DWIN64, -DZLIB_WINAPI in 32-bit build for fixing zlib linkage;
2. Work around MSVC2015 limitation based on lacking the C++14 features (like void as literal type) and document it.

anatoly-spb committed:
Adapt code to compile by MSVC 2015, remove conditional compilation (#105)

Fixes workaround introduced in #101.

Author: @anatoly-spb.
Related to: #17.

Commit:
Merge branch 'app_name_in_tray_menu' of https://github.com/minlexx/kepka into app_name_in_tray_menu

crackedmind commented:
:white_check_mark: [Build kepka 1.0.74-fljrmggr completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.74-fljrmggr) (commit https://github.com/procxx/kepka/commit/a16ae289f2 by @minlexx)
